src/backend/joanie/core/utils/contract.py

```diff
@@ -158,13 +158,14 @@ def generate_zip_archive(pdf_bytes_list: list, user_uuid: str, zip_uuid=None) ->
     return zip_archive_name
 
 
-def order_has_organization_owner(order) -> bool:
+def order_has_organization_owner(order, batch_order) -> bool:
```

Comment:
We can get rid of those batch_order arguments. 
I'll send a proposal commit for convenience.